target/cheri-common/cheri_tagmem.c

```diff
@@ -381,7 +381,10 @@ void *cheri_tagmem_for_addr(RAMBlock *ram, ram_addr_t ram_offset, size_t size)
         return ALL_ZERO_TAGBLK;
 
     uint64_t tag = ram_offset >> CAP_TAG_SHFT;
+#ifndef TARGET_AARCH64
+    // AArch64 seems to use different sizes. Might be worth looking into.
```

Comment:
This sounds like something that could cause issues when booting.

Reply:
I have a feeling when I looked at this I also saw a size of zero...

Reply:
Possibly at least worth putting some of these TODOS / fix-me's as known issues in the tracker.